README.Rmd

```diff
@@ -34,14 +34,13 @@ remotes::install_github("globalgov/qCreate")
 
 ## qPackages
```

Comment:
Are we ready to add the qCreate cheatsheet in here?

Reply:
There are some minor adjustments need for some recent updates, so @EstherPeev will update the cheatsheet on Monday so that we can add it to the PR and, hopefully close it then. Thank you.